### ISSUE TITLE
Tooltip for Glyphs in Enchanter can have Rarity

### ISSUE THREAD

**Issue by Nintai** [ChalkyBrush/Roshpit2-BugTracker#13]
<img width="853" alt="image" src="https://github.com/user-attachments/assets/8eb910f4-ff80-4051-8968-e68f1710b3da">

It seems that you create an item for the tooltip to show, and i can randomly get rarities other than common or whatever.

I would also suggest to give them static rarity like back in rpc1 (not sure about the level requirement)

**Comment by ChalkyBrush** [comment]
yes I've known about this for a while but never got around to fixing it. Also the level requirements
